localization/de/strings.po

```diff
@@ -1957,3 +1957,6 @@ msgstr "Rezept-Karte"
 
 msgid "Group ingredients by their product group"
 msgstr "Zutaten nach Produktgruppen gruppieren"
+
+msgid "Transaction successfully undone"
+msgstr "Vorgang wurde rückgängig gemacht"
```

Comment:
```suggestion
```

That's not needed, needs to be translated on Transifex (see https://github.com/grocy/grocy#localization).